spec/models/stats_spec.rb

```diff
@@ -24,7 +24,27 @@
 
   describe "#assisted_pc" do
     it "returns the correct value" do
-      expect(subject.assisted_pc).to eq(35.4)
+      create(:registration, submitted_at: 1.day.ago.beginning_of_day)
+      create(:registration, submitted_at: 2.day.ago.beginning_of_day)
+      create(:registration, :was_assisted, submitted_at: 3.day.ago.beginning_of_day)
+
+      expect(subject.assisted_pc).to eq(33.3)
+    end
+
+    it "handles no assisted registrations correctly" do
+      (1..3).each do |n|
+        create(:registration, submitted_at: n.days.ago.beginning_of_day)
+      end
+
+      expect(subject.assisted_pc).to eq(0)
+    end
+
+    it "handles all assisted registrations correctly" do
```

Comment:
What does `correctly` means? 
As a developer, I would like to be able to understand how this method `handles` the registration `correctly` when I read the test suite :)

Reply:
Yeah makes sense -- the tests are now clearer about what "handling" means.